src/main/java/az/keytd/expensetracker/controller/IncomeController.java

```diff
@@ -19,4 +21,10 @@ public class IncomeController {
     public List<Income> getAll(){
         return incomeService.getAll();
     }
+
+    @PutMapping ("income")
```

Comment:
with root path it will be like "/income/income" do better name would be "add" so absolute path with root  will be  "income/add"